cmd/forwarder/fixer.go

```diff
@@ -21,48 +22,68 @@ const (
 	maxMsgidLength    = 32
 )
 
-var nilVal = []byte(`- `)
```

Comment:
Mostly because I hate having global package variables when they are only used in one place.

Reply:
With the lower "n" at the beginning it stays a private package. Having it defined here, once, is idiomatic and as Mike suggested, more efficient.

Reply:
I meant global among the entire package main.  It would feel better if you could have constant slice literals.  I'm not going to die on the hill for this though.